src/main/java/com/revature/app/model/Category.java

```diff
@@ -0,0 +1,34 @@
+package com.revature.app.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+
+@Entity
+@Table(name="Categories")
+@AllArgsConstructor @EqualsAndHashCode @Getter @Setter @ToString @NoArgsConstructor
+public class Category {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "category_id")
+	private int category_id;
```

Comment:
For variable names, you might want to use a different naming convention

ex. category_id -> categoryId